website/templates/emails/conference_submitted.txt.mako

```diff
@@ -3,27 +3,26 @@ Hello ${fullname},
 Congratulations! You have successfully added your ${conf_full_name} ${presentation_type} to the Open Science Framework (OSF).
 
 % if user_created:
-Your account on the Open Science Framework has been created. To claim your account, please create a password by clicking here: [ ${set_password_url} ]. Please verify your profile information at [ ${profile_url} ].
+Your account on the Open Science Framework has been created. To claim your account, please create a password by clicking here: ${set_password_url}. Please verify your profile information at: ${profile_url}.
 
 % endif
-Your ${conf_full_name} submission has been added to the Open Science Framework. You now have a permanent, citable URL, that you can share: [ ${node_url} ]. All submissions ${conf_full_name} may be viewed at the following link: [ ${conf_view_url} ].
+You now have a permanent, citable URL, that you can share: ${node_url}. All submissions for ${conf_full_name} may be viewed at the following link: ${conf_view_url}.
 
 % if is_spam:
 Your email was flagged as spam by our mail processing service. To prevent potential spam, we have made your project private. If this is a real project, please log in to your account, browse to your project, and click the "Make Public" button so that other users can view it.
 
 % endif
-Get more from the OSF by enhancing your page with the following:
+Get more from the OSF by enhancing your project with the following:
 
 * Collaborators/contributors to the submission
 * Charts, graphs, and data that didn't make it onto the submission
 * Links to related publications or reference lists
-* Connecting other accounts, like Dropbox, Google Drive, GitHub, figshare and Mendeley via add-on integration. Learn more and read the full list of available add-ons: [ http://osf.io/getting-started/#addons ]
+* Connecting other accounts, like Dropbox, Google Drive, GitHub, figshare and Mendeley via add-on integration. Learn more and read the full list of available add-ons: http://osf.io/getting-started/#addons
 
-To learn more about the OSF, visit [ http://osf.io/getting-started ]!
+To learn more about the OSF, visit: http://osf.io/getting-started
 
-Follow the COS at @OSFramework on Twitter [ https://twitter.com/OSFramework ]
-Like us on Facebook [ https://www.facebook.com/OpenScienceFramework ]
+Follow COS at @OSFramework on Twitter: https://twitter.com/OSFramework
+Like us on Facebook: https://www.facebook.com/OpenScienceFramework
```

Comment:
Sure, so will I be right in making the following changes:

Renaming the file
Structuring it like HTML ( does it need to be wrapped in %def? )
Removing the 2 social links

> On Dec 2, 2015, at 2:08 PM, mfraezz <notifications@github.com> wrote:
> 
> In website/templates/emails/conference_submitted.txt.mako <https://github.com/CenterForOpenScience/osf.io/pull/4639#discussion_r46459222>:
> 
> >  
> > -Follow the COS at @OSFramework on Twitter [ https://twitter.com/OSFramework ]
> > -Like us on Facebook [ https://www.facebook.com/OpenScienceFramework ]
> > +Follow COS at @OSFramework on Twitter: https://twitter.com/OSFramework
> > +Like us on Facebook: https://www.facebook.com/OpenScienceFramework
> Actually, looking at our /mails/ system a bit more, it seems you'd need to rename this file to conference_submitted.html.mako, and structure it like HTML. (The above file may actually have a bug.)
> 
> However, the fact that social links are included in this user confirmation email but not others is a consistency issue. I spoke with @erinmayhood <https://github.com/erinmayhood> and @saradbowman <https://github.com/saradbowman>, and they think these two links should just be removed.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/CenterForOpenScience/osf.io/pull/4639/files#r46459222>.
>